### ISSUE TITLE
Rename Symbol in Vue files convert tabs to spaces

### ISSUE THREAD

**Issue by nicoespeon** [nicoespeon/abracadabra#195]
## Describe the bug

Related to #191

Our custom "rename symbol" in .vue files converts the tab unexpectedly.

## How to reproduce

![2020-11-19_17-50-05](https://user-images.githubusercontent.com/7026091/99722814-fee45980-2a8f-11eb-959a-f7fc2b6d5651.gif)

## Expected behavior

Tabs are preserved, like with other refactorings

## Additional information

- Version of the extension impacted: _v4.11.0_

We are delegating part of the work to Babel's `scope.rename()`. It certainly does the reformat. We should try to either upgrade Babel, or implement it ourselves (and resolve the scope).

**Comment by ZakMiller** [comment]
I created a test to reproduce this issue:

```
   it("preserves tabs", async () => {
      const editor = new InMemoryEditor(`const hello = 'world';
    console.log([cursor]hello);
		const goodMorning = \`Good morning \${hello}!\``);
      jest.spyOn(editor, "delegate").mockResolvedValue(Result.NotSupported);
      jest.spyOn(editor, "askUserInput").mockResolvedValue("aBrandNewName");

      await renameSymbol(editor);

      expect(editor.code).toBe(`const aBrandNewName = 'world';
    console.log(aBrandNewName);
		const goodMorning = \`Good morning \${aBrandNewName}!\``);
    });
```

Note that there are two spaces before `console.log` and two tabs before `const goodMorning`

I also noticed this:
https://github.com/nicoespeon/abracadabra/blob/a7104b83c210d9f49428858dace5b8fe4d7d89f2/src/ast/transformation.ts#L162

When I comment that out the test passes! But a couple other tab related tests fail (the refactorings just don't work).

[Related](https://github.com/nicoespeon/abracadabra/commit/172fa4f062695baa4e25d874702c31c337be8854)

**Comment by nicoespeon** [comment]
Interesting, thanks for the test case. That will be helpful!

**Comment by DangerousDetlef** [comment]
I really like your extension since I'm in need of something to refactor my variable names in .vue files, but this is really the issue that keeps me from actively using it since we use tabs instead of spaces in our company (I know, they also eat babies as far as I'm concerned 😁 ). Would be great to see a fix soon.

**Comment by nicoespeon** [comment]
@DangerousDetlef yeah, I understand the frustration, sorry about that 😁 

Thanks for raising it up. I gave it another try today and it turned out this refactoring wasn't using the regular pattern others are using. Thus, it wasn't benefiting of the logic that is preserving tabs or spaces—we had to solve that in the past already!

Anyway, this is fixed now. I'll ship that in the next release, in the coming days 👍
